csaf_2.1/prose/edit/src/safety-security-and-data-protection.md

```diff
@@ -31,4 +31,16 @@ Moreover, it SHALL be treated as unsafe (user) input.
   > Additional, supporting mitigation measures like retrieving only CSAF documents from trusted sources and check their integrity and
   > signature before parsing the document SHOULD be in place to reduce the risk further.
 
+The distribution requirements of CSAF data allow to specify domains as the value of the HTTP header `Access-Control-Allow-Origin`.
+While a wildcard (`*`) as header value usually prevents implementing browsers from sending credentials during the CORS request,
+the restriction to specified domains often enables sending credentials.
+Allowing several specified domains results in using dynamics on the server, which can widen the attack surface slightly by more code and configuration.
```

Comment:
Maybe the "slightly" quantifier could go, or even better we could use another sentence to weight the consequences in preparation of the following "therefore ..." conclusion?

Reply:
Removed the "slightly" in [8c119fc](https://github.com/oasis-tcs/csaf/pull/903/commits/8c119fc6b975a99952a5cc4763aa9c492412e961).

Reply:
Do you have a suggestion for the sentences?

Reply:
I would now just keep it as it is - finishing mode ...